src/Callback/Success.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FunctionComponent } from "react"
+import { useTranslation } from "react-i18next"
+import { ScrollView, StyleSheet, Image } from "react-native"
+import { useNavigation } from "@react-navigation/native"
+
+import { GlobalText, Button } from "../components"
+import { Typography, Spacing } from "../styles"
+import { Stacks } from "../navigation"
+import { Images } from "../assets"
+
+const Success: FunctionComponent = () => {
```

Comment:
This probably wants `useStatusBarEffect("light-content", Colors.headerBackground)`